dataland-frontend/src/components/generics/ViewFrameworkBase.vue

```diff
@@ -74,27 +74,27 @@
 </template>
 
 <script lang="ts">
-import FrameworkDataSearchBar from "@/components/resources/frameworkDataSearch/FrameworkDataSearchBar.vue";
-import MarginWrapper from "@/components/wrapper/MarginWrapper.vue";
 import BackButton from "@/components/general/BackButton.vue";
-import TheHeader from "@/components/generics/TheHeader.vue";
+import TheFooter from "@/components/general/TheFooter.vue";
 import TheContent from "@/components/generics/TheContent.vue";
-import AuthenticationWrapper from "@/components/wrapper/AuthenticationWrapper.vue";
+import TheHeader from "@/components/generics/TheHeader.vue";
 import CompanyInformation from "@/components/pages/CompanyInformation.vue";
-import PrimeButton from "primevue/button";
+import FrameworkDataSearchBar from "@/components/resources/frameworkDataSearch/FrameworkDataSearchBar.vue";
+import AuthenticationWrapper from "@/components/wrapper/AuthenticationWrapper.vue";
+import MarginWrapper from "@/components/wrapper/MarginWrapper.vue";
 import { ApiClientProvider } from "@/services/ApiClients";
-import { defineComponent, inject, ref } from "vue";
-import Keycloak from "keycloak-js";
-import { assertDefined } from "@/utils/TypeScriptUtils";
-import Dropdown, { DropdownChangeEvent } from "primevue/dropdown";
-import { humanizeString } from "@/utils/StringHumanizer";
 import { ARRAY_OF_FRAMEWORKS_WITH_UPLOAD_FORM, ARRAY_OF_FRAMEWORKS_WITH_VIEW_PAGE } from "@/utils/Constants";
-import TheFooter from "@/components/general/TheFooter.vue";
-import { DataMetaInformation, DataTypeEnum } from "@clients/backend";
 import { checkIfUserHasUploaderRights } from "@/utils/KeycloakUtils";
+import { humanizeString } from "@/utils/StringHumanizer";
+import { assertDefined } from "@/utils/TypeScriptUtils";
+import { DataMetaInformation, DataTypeEnum } from "@clients/backend";
+import Keycloak from "keycloak-js";
+import PrimeButton from "primevue/button";
+import Dropdown, { DropdownChangeEvent } from "primevue/dropdown";
+import { defineComponent, inject, ref } from "vue";
 
-import OverlayPanel from "primevue/overlaypanel";
 import SelectReportingPeriodDialog from "@/components/general/SelectReportingPeriodDialog.vue";
+import OverlayPanel from "primevue/overlaypanel";
```

Comment:
Why did the order of the imports change?
If this happened accidentally, let's revert it.

Reply:
I have an "Optimize imports" functionality in VS Code which not only orders alphabetically but removes unused imports. The order doesn't affect compilation in any way.